### ISSUE TITLE
Research projects may want to keep results data private for a given time

### ISSUE THREAD

**Issue by teleyinex** [Scifabric/pybossa#218]
**Problem**:  researchers would worry about a variety of things - from people trying to actively manipulate the data based on what others have put in, to people using the data without discussing it with the researchers - if everyone had access to it from the beginning.
**Solution**: Give the owners of the application an option to hide the data from the web access and API

@PyBossa  This issue could be really interesting to discuss about it, as this option could led to projects where researchers will not release any data after a given period of time, but at the same time not offering this feature could cause that other researchers will never consider the platform as they cannot control the data at all. 

I guess this is question where we need to talk and discuss with the scientists and see how we can agree in a mid term solution, where the scientists will have control in the first time over the data for publishing papers, etc. but at the same time the contributions made by the volunteers should be respected and returned to them in one way or another. For example, PyBossa will provide an option to keep the data private with a maximum deadline of one year. After that deadline is met, the data will be made public, so the user contributions will not be lost.

This is really important, because, IMHO, there is no a simple solution for this. We may find researchers that will find ok to share the data from the beginning without problems, but there will be others that will be reluctant but will want to use the platform. If they do not have a bit of control about their data, they will go away and we will miss the opportunity to explain to them why is important to share the data since the very first moment. I guess PyBossa could be used as a tool to get researchers more open about open-science and open-data if we provide a good set of tools for all of them.

We also may need to consider that some experiments run by the researchers will need actually to not show any data until all the tasks are completed, so we need to basically discuss this a bit more with scientists and researchers.

In summary, I would like to open a discussion about this issue and how do you think we should approach it.
